dev/TabView/TabViewAutomationPeer.idl

```diff
@@ -0,0 +1,12 @@
+﻿namespace MU_XAP_NAMESPACE
+{
```

Comment:
FWIW now that these files have namespace (they didn't for a long time), you can have the peers and the implementation in the same IDL file. Not sure if this is where we want to start that pattern but I think less files is easier and makes the automation peer type feel a bit less like the red-headed step child.

Reply:
I like that pattern MUCH better. Done.